_virtual_machines/fedora-ws-35.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Fedora Workstation 35
+architecture: ARM64
+memory: 4 GiB
+disk: 5 GiB
```

Comment:
Should this be `64 GiB` since that's the default?

Reply:
Wasn't sure if that was how much space the install used or the "recommended" or "default". Can make it whatever.